osmosis-1/osmosis.zone_assets.json

```diff
@@ -1631,6 +1631,13 @@
       "path": "transfer/channel-326/stusomm",
       "osmosis_main": true,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "cosmoshub",
```

Comment:
this should be corrected to chain_name: fxcore now

Reply:
resolved